src/modm/platform/uart/stm32/module.lb

```diff
@@ -25,23 +25,18 @@ class Instance(Module):
     def prepare(self, module, options):
         module.depends(":platform:uart")
 
-        module.add_option(
-            BooleanOption(
-                name="buffered",
-                description="",
-                default=True))
         module.add_option(
             NumericOption(
                 name="buffer.tx",
                 description="",
-                minimum=1, maximum=2 ** 16 - 2,
+                minimum=0, maximum=2 ** 16 - 2,
                 default=250))
```

Comment:
Should this be zero too? It would be consistent and I think 250 bytes of buffer is fairly arbitrary. All our BSPs explicitly set this buffer higher. But it's not backwards compatible…